code/game/ai_dmq3.c

```diff
@@ -3567,7 +3597,9 @@ void BotAimAtEnemy(bot_state_t *bs) {
 	//
 	if (wi.number == WP_MACHINEGUN ||
 			wi.number == WP_SHOTGUN ||
+			wi.number == WP_NAILGUN ||
```

Comment:
Actually this was only meant for instant hit weapons, can you explain why you added the nailgun here?